### PR TITLE
Grammar fixes for tense, punctuation and wording.

### PR DESCRIPTION
**Pull request by polizz**
Great guide, thanks for assembling all these resources. Just a few edits that clarify some of the language!

### CODE REVIEW

File: preparing/README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ Java is a decent choice too but having to constantly declare types in your code
...
 
-### Revise your CS101
+### Review your CS101
 
-If you have been out of college for a while, it will be highly advisable to revise the CS fundamentals — Algorithms and Data Structures. Personally I prefer to revise as I practiced so I scanned through my college notes and revised the various algorithms as I worked on the algorithm problems from LeetCode and Cracking the Coding Interview.
+If you have been out of college for a while, it is  highly advisable to review CS fundamentals — Algorithms and Data Structures. Personally, I prefer to review as I practice, so I scan through my college notes and review the various algorithms as I work algorithm problems from LeetCode and Cracking the Coding Interview.
```
Comment (yangshun) [inline preparing/README.md]: Extra space after `is`.

**Comment by yangshun** [comment]
Thank you! There's just one minor issue and I can merge this after it is fixed (:

File: preparing/README.md (hunk 1 above)
Comment (yangshun) [reply preparing/README.md]: I think it should be `work on` instead of just `work`.

Comment (polizz) [reply preparing/README.md]: Good eye!

**Comment by yangshun** [comment]
Thank you for your contribution! More PRs are welcome! It's a good way to contribute to [Hacktoberfest](https://hacktoberfest.digitalocean.com/) 😎

**Pull request merged**